### PR TITLE
Updating handbook for docs and articles.

### PR DESCRIPTION
**Pull request by rachelandrew**
Adding some clarification around what goes where.

### CODE REVIEW

File: site/en/docs/handbook/where-to-publish/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+If the thing mentioned in your blog post changes, then post a new blog post to alert developers to the updates, rather than update an old post, which will get very little visibility and is potentially confusing.
+
+## Articles
+
+Content placed in [articles](/articles/) should be considered _evergreen_. We might revisit and update it as things change. This should be engaging developer focused content, and include [tutorials](https://documentation.divio.com/tutorials/), [explanations](https://documentation.divio.com/explanation/), and [how-tos](https://documentation.divio.com/how-to-guides/). 
+
+## Example
+
+On launching a new API as an Origin Trial. Create:
```
Comment (mihajlija) [inline site/en/docs/handbook/where-to-publish/index.md:36]:
```suggestion
On launching a new API as an origin trial. Create:
```

File: site/en/docs/handbook/where-to-publish/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- Reference documentation under `/docs`.
+- A short blog post announcing the Origin Trial, and linking to the documentation.
```
Comment (mihajlija) [inline site/en/docs/handbook/where-to-publish/index.md:39]:
```suggestion
- A short blog post announcing the origin trial, and linking to the documentation.
```

File: site/en/docs/handbook/how-to/add-an-article/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## Clone the sample post
+
+Duplicate the example post located in `site/en/articles/_example/` and
+rename the folder to match whatever slug you want to use for your url.
+
+Posts support markdown syntax with some additional features and shortcodes.
+
+## Metadata
+
+Each `index.md` file starts with YAML Front Matter. See
```
Comment (mihajlija) [inline site/en/docs/handbook/how-to/add-an-article/index.md:21]:
```suggestion
Each `index.md` file starts with YAML front matter. See
```

File: site/en/docs/handbook/how-to/add-an-article/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+`site/en/articles/_example/` for documentation of all supported metadata.
+
+Articles and blog posts share the same template and are identical other than location at the moment.
+
+### Hero images
+
+Hero images should be at least 1920 x 960. Use jpg unless you have a specific
+need for png or svg. Our image CDN will handle compressing and converting the
+image to newer formats such as webp so you don't have to do that in advance.
```
Comment (mihajlija) [inline site/en/docs/handbook/how-to/add-an-article/index.md:30]:
```suggestion
Hero images should be at least 1920 x 960. Use JPG unless you have a specific
need for PNG or SVG. Our image CDN will handle compressing and converting the
image to newer formats such as WebP so you don't have to do that in advance.
```

File: site/en/docs/handbook/where-to-publish/index.md (hunk 1 above)
Comment (mihajlija) [reply site/en/docs/handbook/where-to-publish/index.md:36]: Good one to add to the word list.

File: site/en/docs/handbook/how-to/add-an-article/index.md (hunk 4 above)
Comment (rachelandrew) [reply site/en/docs/handbook/how-to/add-an-article/index.md:30]: This is duplicated from the other how-tos, which likely date from the launch of this site. I should go update those in another PR - thanks!

**Pull request merged**